go.mod

```diff
@@ -7,9 +7,10 @@ toolchain go1.22.3
 require (
 	github.com/cloud-bulldozer/go-commons v1.0.15
 	github.com/google/uuid v1.6.0
-	github.com/kube-burner/kube-burner v1.9.8
+	github.com/kube-burner/kube-burner v1.10.0
```

Comment:
Let's remove references to the `clusterMetadata` document in the docs https://kube-burner.github.io/kube-burner-ocp/latest/#cluster-metadata

Reply:
Done Removed.